### DISCUSSION TITLE
draw vertical histogram

### DISCUSSION THREAD

**Discussion by mattou78400** [tradingview/lightweight-charts#778] [Q&A] [answered]
Hey everyone,

I was wondering if anyone managed to draw vertical histograms under the candles? see screenshot ex: 
![image](https://user-images.githubusercontent.com/39678117/123066886-c2efe900-d410-11eb-9e25-83c4fb8a1788.png)

I'd like to show some data like this, but can't figure out at all how to draw such thing on lightweight chart
Thanks a lot!

**Comment by timocov** [comment, marked as answer]
No, this is impossible to do with lightweight-charts and we don't have any plans to implement it. But probably after #408  you will be able to implement it yourself.

**Reply by mattou78400** [reply]
Thanks a lot!
Is there any ETA by any chance? :)

**Reply by timocov** [reply]
No, we don't have any ETA.

**Reply by emorling** [reply]
Is there any way to get the x,y of the price along the vertical side, so that this could be plotted with absolute positioning outside of the chart?
